src/main/java/apoc/meta/Meta.java

```diff
@@ -443,12 +446,14 @@ private void collectStats(Collection<String> labelNames, Collection<String> relT
                 }
             }
             try (ResourceIterator<Node> nodes = db.findNodes(label)) {
```

Comment:
Modifica il pezzo di codice in questo modo, in questo ricalcoliamo `sample` solo quando è entrato nell'if e non ad ogni chiamata dell'if che potrebbe essere dispendioso e non beccare mai valore `true` (visto che il numero oscillerebbe continuamente):

```java
long sample = config.getSampleForLabel(label.name())
try (ResourceIterator<Node> nodes = db.findNodes(label)) {
	int count = 1;
	while (nodes.hasNext()) {
		Node node = nodes.next();
		if(count++ % sample == 0) {
			sample = config.getSampleForLabel(label.name());
			addRelationships(metaData, nodeMeta, labelName, node, relConstraints);
			addProperties(nodeMeta, labelName, constraints, indexed, node, node);
		}
	}
}
```